packages/salesforcedx-vscode-lwc/src/service/devServerService.ts

```diff
@@ -49,4 +50,12 @@ export class DevServerService {
       console.log('lwc dev server was not running');
     }
   }
+
+  public getBaseUrl(): string {
```

Comment:
Instead of reading from constants, we set and retrieve server url info from here.